### DISCUSSION TITLE
How to Configure and Check Feed Sync Timing in Greenbone Docker Container?

### DISCUSSION THREAD

**Discussion by devMarc5598** [immauss/openvas#300] [Q&A] [answered]
Hi everyone, I need some help regarding the feed sync process.
1) **Viewing the Feed Sync Time**: I’d like to know where I can check the specific time when the feed sync happens within the Docker container. Is there a log or configuration file that shows when the last sync occurred and when the next one is scheduled?
2) **Setting a Manual Feed Sync Time**: Additionally, I’d like to configure or edit a custom time for the feed sync to occur. Is there a way to manually specify when the sync should happen (e.g., via a cron job or a system setting in the container)?

Any guidance on how to manage these tasks would be greatly appreciated! Thanks in advance for your help.

**Comment by immauss** [comment, marked as answer]
The feeds do not update automatically. 
As long as you didn't start the container with "-e SKIPSYNC=true", restarting the contianer will updated the feeds. 

you can also run the sync.sh script in the container to initiate a sync. 

```
docker exec <container name> /scripts/sync.sh 
```

If you wanted to setup a cron on the host machine to run that command, that would work as well to give you a scheduled sync. 

-Scott
